react/components/ProductList/index.tsx

```diff
@@ -12,6 +13,14 @@ interface Props {
 const CSS_HANDLES = ['productList', 'productListItem']
 
 const ProductList: FC<Props> = ({ products }) => {
+  const { bagsIDs } = useGetBagsSgrIDs()
+
+  // Put all the bags at the end of the products list
```

Comment:
Care este scopul?

Reply:
Este cerinta in plus de la Diana. Pungile sa apara la final in lista de produse.